### PR TITLE
[ES6 class] Update MesosSummaryStore to es6 class

### PR DESCRIPTION
**Pull request by orlandohohmeier**
---
ℹ️   _This branch depends on features that will be introduced with #275, thus it will fail until the respective branch is merged._

---
Update store to es6 class – this is part of series of PRs to update all the stores as discussed in https://github.com/dcos/dcos-ui/pull/246/files/b4b52fcdb5f899fd74f9bd68cfa3d216177fcecf#r64962494

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: src/js/events/MesosSummaryActions.js [hunk 1]
```diff
@@ -84,12 +85,25 @@ var MesosSummaryActions = {
     Config.getRefreshRate(),
     function (resolve, reject) {
 
-      return function (timeScale) {
+      return function (timeScale, placeholder = false) {
+        if (placeholder) {
+          AppDispatcher.handleServerAction({
+            type: REQUEST_SUMMARY_PLACEHOLDER,
```
Comment (rcorral) [inline src/js/events/MesosSummaryActions.js]: I think we need a better solution for this.

File: src/js/stores/MesosSummaryStore.js [hunk 2]
```diff
@@ -38,12 +37,7 @@ function startPolling() {
       } else {
         // If not active, push null placeholder. This will ensure we maintain
         // history when navigating back, for case where history server is down.
-
-        // Use {silent: true} Because we only want to push a summary on the stack without side
-        // effects (like re-rendering etc). The tab is out of focus so we
-        // don't want it to do any work. It only matters that there is
-        // appropriate history when we return focus to the tab.
-        MesosSummaryStore.processSummaryError({silent: true});
+        MesosSummaryActions.fetchSummary(undefined, true);
```
Comment (rcorral) [inline src/js/stores/MesosSummaryStore.js]: Can we instead just do this at the bottom of the file: `MesosSummaryStoreInstance = new MesosSummaryStore();`
and call `MesosSummaryStoreInstance.processSummaryError({silent: true});` just like we were doing before?

I don't like the idea of calling a method on an Actions file that we know will return faked data as if it were making a call to an API. I don't believe that's the best layer to fake the data.

Comment (orlandohohmeier) [inline src/js/stores/MesosSummaryStore.js]: I'll change it back to the `processSummaryError({silent: true});` method call as the refactoring is actually not in the scope of this PR. But I'd like to discuss this in our next meeting, as I think that actions, following the flux pattern, are the right place to do that kind of stuff.

/cc @mlunoe

Comment (mlunoe) [inline src/js/stores/MesosSummaryStore.js]: Sounds good!

**Pull request merged**